demo/docbot/cluster_bootstrap.py

```diff
@@ -168,43 +171,64 @@ def initialize_connector(client: MLClient) -> None:
         print("Connector 'Cohere Connector' already exists. Skipping initialization.")
 
 
-def initialize_model(client: MLClient) -> None:
+def initialize_model(client: MLClient, model_name: str, model_descp: str) -> None:
     """
+    Initialize a model in OpenSearch.
+
     Args:
-        Client: OpenSearch
+        client (MLClient): The OpenSearch ML client.
+        model_name (str): The name of the model to initialize.
+        model_descp (str): The description of the model.
+
     Returns:
-        returns None if model was created or is exists else raises Exception
+        None if model is initialized successfully or already initialized else raises Exception
+
     """
-    # Define the model data. For now, placeholders are used for model_group_id and connector_id.
-    model_data = {
-        "name": "embed-english-v2.0",
+    model_meta = {
+        "name": model_name,
         "function_name": "remote",
-        "description": "test model",
+        "description": model_descp,
         "model_group_id": MODEL_STATE["model_group_id"],
         "connector_id": MODEL_STATE["connector_id"]
     }
 
-    # Check if the model already exists
+    if model_name == "command-nightly":
+        model_meta["parameters"] = {"model": "command-nightly"}
+
+    if model_exists(client, model_name):
+        print("Model " + model_name + " already exists. Skipping initialization.")
+        return
+
+    response = client.register_connector_model(model_meta_json=model_meta)
+    if isinstance(response, str):
+        if (model_name == "command-nightly"):
+            MODEL_STATE["LLM_model_id"] = response
+        else:
+            MODEL_STATE["model_id"] = response
+        print("Model " + model_name + " initialized successfully!")
+    else:
+        raise Exception("Failed to initialize model.")
+
+
+def model_exists(client: MLClient, model_name: str) -> bool:
+    """
+    Args:
+        client (MLClient): The OpenSearch ML client.
+        model_name (str): The name of the model to check.
+    Returns:
+        returns True if the model exists else False
+    """
     existing_models = client.search_model({
         "query": {
             "match": {
-                "name": "embed-english-v2.0"
+                "name": model_name
             }
         }
     })
-
     if existing_models["hits"]["total"]["value"] > 0:
-        print("Model 'embed-english-v2.0' already exists. Skipping initialization.")
         MODEL_STATE["connector_id"] = existing_models["hits"]["hits"][0]["_id"]
-    else:
-        # Register (and deploy) the model
-        response = client.register_connector_model(model_meta_json=model_data)
-
-        if isinstance(response, str):
-            print("Model 'embed-english-v2.0' initialized successfully!")
-        else:
-           raise Exception("Failed to initialize model.")
-
+        return True
```

Comment:
Can we lift this into the parent function so this will either return model ID or None and then set from the calling function.